playground/module-resolution/__tests__/module-resolution.spec.ts

```diff
@@ -106,5 +112,13 @@ describe.runIf(!isBuild)('module resolution', async () => {
 			const result = await getTextResponse('/@alias/test');
 			expect(result).toBe('OK!');
 		});
+
+		test('imports from a non configured aliased package', async () => {
```

Comment:
What does "non configured" actually mean?

Reply:
I think you mean non-aliased, right?

Reply:
I've updated the code and test to clarify this 🙂 

Having it as an alias, which just so happen not to be correctly configured is probably the wrong mindset, I've updated it to be a non-existing package instead, so that the testing is regarding a user error in which a non existing package gets imported

Please have another look 🙂